.ci/pytorch/multigpu-test.sh

```diff
@@ -44,8 +44,6 @@ time python test/run_test.py --verbose -i distributed/_tensor/test_dtensor_compi
 time python test/run_test.py --verbose -i distributed/test_device_mesh
 
 # DTensor/TP tests
-time python test/run_test.py --verbose -i distributed/tensor/parallel/test_ddp_2d_parallel
-time python test/run_test.py --verbose -i distributed/tensor/parallel/test_fsdp_2d_parallel
 time python test/run_test.py --verbose -i distributed/tensor/parallel/test_tp_examples
```

Comment:
you will also need to add the new test file here

Reply:
> you will also need to add the new test file here

new tests are included in [test_2d_composability](https://github.com/pytorch/pytorch/pull/129802/files#diff-34b5ad4462d9fccf96dcd7c22fe3f3eaa7c23cb60886281be02f00d32b5d4f4aR54)